support/awsutil/sg.go

```diff
@@ -227,3 +229,50 @@ func GetSecurityGroupById(ec2Client ec2iface.EC2API, id string) (*ec2.SecurityGr
 	}
 	return describeSGResult.SecurityGroups[0], nil
 }
+
+func UpdateResourceTags(ec2Client ec2iface.EC2API, resourceID string, create, remove map[string]string) error {
+	// If we have anything to create or update
+	if len(create) > 0 {
+		createTagsInput := MapToEC2Tags(create)
+		input := &ec2.CreateTagsInput{
+			Resources: []*string{aws.String(resourceID)},
+			Tags:      createTagsInput,
+		}
+
+		// Create/Update tags in AWS.
+		if _, err := ec2Client.CreateTags(input); err != nil {
+			return errors.Wrapf(err, "failed to create tags for resource %q: %+v", resourceID, create)
```

Comment:
Does this return an error when the tags you're trying to create already exist? If so, we should likely ignore.